test/presenters/supergroups/news_and_communications_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
+include RummagerHelpers
+
+describe NewsAndCommunications do
+  let(:taxon_id) { '12345' }
+  let(:news_and_communications_supergroup) { NewsAndCommunications.new }
+
+  describe '#document_list' do
+    it 'returns a document list for the services supergroup' do
```

Comment:
Typo in the test name. The supergroup should be "news and communications".

Reply:
Thanks for catching that. There are other as I was doing some copy and pasting...... Fixed now